File: Week_01/Day_04/Task_01.cpp
//Write a program to find the largest among three numbers. 
#include<iostream>
using namespace std;
int main(){
	float n1,n2,n3,largest;
	cout<<"Enter 3 Numbers : ";
	cin>>n1>>n2>>n3;
	if(n1>n2 && n1>n3){
		cout<<n1<<" Is Largest"<<endl;
	}else if(n2>n1 && n2>n3){
		cout<<n2<<" Is Largest"<<endl;
	}else{
		cout<<n3<<" Is Largest"<<endl;
	}
	return 0;
}

File: Week_02/Day_02/Task_01.py
#Write a program to swap two numbers without using a third variable.

num1 = float(input("Enter First Number : "))
num2 = float(input("Enter Second Number : "))

#lets suppose we have num1 as 5 and num2 as 3
num1 = num1 + num2  # 5 + 3 --> 8... now, num1 = 8
num2 = num1 - num2  # 8 - 3 --> 5... then, num1 which is 8 - num2 which is 3
num1 = num1 - num2  # 8 - 5 --> 3... then, num1 which is 8 - num2 which is now 5 (after 8-3)

print("After Swapping : ",num1,num2)
